pallets/liquidity-pools/src/tests/inbound.rs

```diff
@@ -181,10 +183,8 @@ mod handle_tranche_tokens_transfer {
 				}
 			));
 
-			let origin = EVM_DOMAIN_ADDRESS.domain().into_account();
-			assert_eq!(Tokens::balance(TRANCHE_CURRENCY, &origin), 0);
-
```

Comment:
Should it not be 0 after transferring the tranche tokens?

Reply:
Yes and no. The test setup was unrealistic because it tested receiving a tranche token transfer from EVM chain 1 to EVM chain id 1. Fixed in [c75a38d](https://github.com/centrifuge/centrifuge-chain/pull/1964/commits/c75a38de95ca797494a4d3621b3dd826af42b4aa)

Reply:
Thanks for fixing it!